app/views/admin/workshops/show.html.haml

```diff
@@ -47,6 +47,9 @@
           %strong.label.round.primary #{@workshop.venue.seats} student spots, #{@workshop.venue.coach_spots} coach spots
           - unless @workshop.spaces?
             %label.label.round.warning Full
+          %p.description
+            %br
+            = @workshop.description.html_safe
```

Comment:
So this is the same security hole ... you need to sanitize this as well. Same fix. But you can see the issue here every time you use description you are having to think about security. Rather than sanitizing it within the WorkshopPresenter.description - so you never have to think about it (which is my future pull request).